src/sqlalchemydiff/inspection/inspectors.py

```diff
@@ -209,7 +209,7 @@ def _format_unique_constraint(self, inspector: Inspector, table_name: str) -> li
             if not name:
                 name = f"unique_{table_name}_{'_'.join(constraint.get('column_names'))}"
             constraint["name"] = name
-        return result
+        return cast(list[dict], result)
```

Comment:
Is this because ty does not like the type of result?

Reply:
This passes my 'ty' check
```
from sqlalchemy.engine.reflection import ReflectedUniqueConstraint

...

def _format_unique_constraint(self, inspector: Inspector, table_name: str) -> list[ReflectedUniqueConstraint]:
    ...
    return result
```

Reply:
yup 👍

Reply:
cheers :)